app/services/oregon_digital/triple_powered_service.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module OregonDigital
+  # Triple Powered Service to fetch out data from uri and grouped as 'label$uri'
+  class TriplePoweredService
```

Comment:
Maybe rename to `LabelFetchService` to better match OD2 naming scheme & `LabelParseService`
(Don't forget to rename file)

Reply:
Yes I will rename it to fit the scheme. I'm currently rebuilding the image trying to fix the `docker-compose test` but will update it once got everything to pass the test with the updated on there.